src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -4094,7 +4093,7 @@ namespace ConvectionCoefficients {
                 "[m], Inside Model Assignment, Inside Height [m], Inside Perimeter Envelope [m], Inside Hydraulic Diameter [m], Window Wall Ratio, "
                 "Window Location, Near Radiant {{Yes/No}}, Has Active HVAC {{Yes/No}}\n");
             print(outputFiles.eio, Format_900); // header
-            for (SurfLoop = 1; SurfLoop <= TotSurfaces; ++SurfLoop) {
+            for (int SurfLoop : DataSurfaces::AllSurfaceListReportOrder) {
```

Comment:
One of may places where `DataSurfaces::AllSurfaceListReportOrder` is used to preserve the same surface order for 90% of output reports. Some lesser used outputs like the shd file, are more complicated and do have some order differences.

Reply:
It's nice that you did this with a range based loop.  Your statement makes sense, I figured there would be a number of places that you'd want to persist order.